### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/tutorial/create.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#669] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/tutorial/create.html

此教程将指引你创建一个 VuePress Theme Hope 项目。

https://theme-hope.vuejs.press/zh/cookbook/tutorial/create.html

**Comment by yeluo244** [comment]
nice
